examples/contrib/gp/sv-dkl.py

```diff
@@ -138,14 +138,13 @@ def cnn_fn(x):
                         help='number of epochs to train (default: 10)')
     parser.add_argument('--lr', type=float, default=0.01, metavar='LR',
                         help='learning rate (default: 0.01)')
-    parser.add_argument('--no-cuda', action='store_true', default=False,
-                        help='disables CUDA training')
+    parser.add_argument('--cuda', action='store_true', default=False,
```

Comment:
Did you mean change the behavior, or do you want to preserve behavior and change the default value to `True`? (I'm fine changing behavior, I just want to confirm your intent)

Reply:
i meant to change it, since for all examples, default is cpu unless specified otherwise. sorry should have called that out.